### PR TITLE
[bug 1221303] Update pixel campaign id's on /firefox/os/2.5/

### PR DESCRIPTION
**Pull request by alexgibson**

### CODE REVIEW

**Comment by alexgibson** [comment]
@malena r?

**Comment by alexgibson** [comment]
Malena is on PTO today - @jpetto or @schalkneethling r?

**Comment by alexgibson** [comment]
For the record, I'm [going by this for direction](https://bugzilla.mozilla.org/show_bug.cgi?id=1221303#c6).

File: media/js/firefox/os/firefox-os-2.5.js [hunk 1]
```diff
@@ -8,22 +8,23 @@
     $('.cta-button').on('click', function(e) {
         if (!window._dntEnabled()) {
```
Comment (jpetto) [inline media/js/firefox/os/firefox-os-2.5.js]: Would it be better to put this `if` statement around the event listener? Or am I over-thinking it?

Comment (alexgibson) [inline media/js/firefox/os/firefox-os-2.5.js]: Could do yeah, why not :)

**Comment by jpetto** [comment]
I think your inference on which tag belongs on which CTA is correct, based on the little detail we have. :+1:

**Comment by alexgibson** [comment]
Thanks @jpetto updated. Let me know if looks good and I'll squash

**Comment by jpetto** [comment]
Looking good. Squarsh time!

**Comment by alexgibson** [comment]
Squished, merge at will :anchor:

**Pull request merged**